package.json

```diff
@@ -118,6 +118,7 @@
     "gulp-nop": "0.0.3",
     "gulp-rename": "2.0.0",
     "gulp-sourcemaps": "3.0.0",
+    "gulp-typescript": "6.0.0-alpha.1",
```

Comment:
We're moving away from `gulp` because it's outdated and not being maintained. Let's just use `typescript` in whatever way possible.

Reply:
See https://github.com/ampproject/amphtml/pull/33247#discussion_r593504160 for an example of how to directly use `typescript`.